scvi/module/base/_base_module.py

```diff
@@ -69,19 +92,19 @@ def _get_dict_sum(dictionary):
         return total
 
     @property
-    def loss(self) -> torch.Tensor:
```

Comment:
Might be overkill, but at the loss recorder level it would be nice if the values were just stored agnostic to the storage type (tensor vs jnp array).